websubhub-ballerina/build.gradle

```diff
@@ -276,12 +298,18 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":websubhub-native:build"
-ballerinaTest.dependsOn ":websubhub-native:build"
+ballerinaBuild.dependsOn ":${packageName}-native:build"
```

Comment:
Shall we move all the task dependencies here? Otherwise this is harder to read.

Reply:
* Refactored the task-dependencies in build.